wolox-books/src/services/localServiceStorage.js

```diff
@@ -0,0 +1,22 @@
+export function setValue(key, value) {
```

Comment:
I think it should be `localStorageService` :thinking: :stuck_out_tongue_closed_eyes: